content/en/events/isc-2025.md

```diff
@@ -5,14 +5,66 @@ date: 2025-11-13
+# A Unique Global Experience
+This summit represents a first-of-its-kind opportunity to connect with the InnerSource community across three continents simultaneously, offering unprecedented reach and impact for sponsors.
+# Connect with a Global InnerSource Audience
+Participants will join both in-person across our three city hubs and via livestream, creating a truly international gathering of InnerSource practitioners, leaders, and enthusiasts.
+# Join Leading Technology Organizations
+Each year, top companies and organizations in tech sponsor our events. We invite you to join them in 2025 for exceptional exposure and networking opportunities with the global InnerSource community.
+# World-Class Venues in Tech Innovation Centers
+Our three host cities—Yokohama, Berlin, and New York—represent major global hubs for technology and innovation, placing your brand at the center of the conversation.
+# Fair Pricing with Exceptional ROI Potential
+We prioritize fair pricing structures designed to maximize your return on investment. Our multi-location format offers the unique opportunity to target specific regional markets or achieve global reach in a single event.
+# Celebrate a Decade of InnerSource Innovation
+By sponsoring, you'll be supporting and connecting with the community that's been driving InnerSource adoption and best practices for the past decade.
+For more information about sponsorship opportunities, contact Addie Girouard, Director of Sponsorship at Addie@InnerSourceCommons.org
+
+### Call for Presentation Now Open
+
+This special 24-hour global event celebrates our 10th anniversary with sessions scheduled across three time zones. We warmly welcome proposals from around the world, as sessions will be hosted in Yokohama (Japan), Berlin (Germany), and New York (USA). To reach the widest possible audience, the community will have the option of attending in-person and/or virtually.
+
+Each presentation will be delivered live, lasting approximately 20 minutes, followed by a brief discussion. Speakers should plan to present live and, if possible, in person at one of our event locations.
+
+Proposal submissions are due by **5PM GMT on Monday, July 15th**.
+
+InnerSource Commons presentations will be any topic that is related to the practice of InnerSource. Here are a few topic ideas, others are welcome:
+
+Getting Started with InnerSource
+InnerSource Best Practices
+InnerSource Culture
+InnerSource Return on Investment and Convincing Leadership
+InnerSource Governance
+InnerSource Legal and Finance
+InnerSource Processes
+InnerSource Metrics and Measurement
+InnerSource Patterns
+InnerSource Tools & Technology
+Incentivizing InnerSource Collaboration
+InnerSource Source Program Offices (ISPOs)
+InnerSource Commons / Building a Global InnerSource Community
```

Comment:
Consecutive lines display as a single paragraph in markdown by default.

![image](https://github.com/user-attachments/assets/474e1fb8-135e-4426-8b7e-3025ff927b7e)

Can make this as a list to fix that.

```suggestion
* Getting Started with InnerSource
* InnerSource Best Practices
* InnerSource Culture
* InnerSource Return on Investment and Convincing Leadership
* InnerSource Governance
* InnerSource Legal and Finance
* InnerSource Processes
* InnerSource Metrics and Measurement
* InnerSource Patterns
* InnerSource Tools & Technology
* Incentivizing InnerSource Collaboration
* InnerSource Source Program Offices (ISPOs)
* InnerSource Commons / Building a Global InnerSource Community
```